tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -250,7 +172,7 @@ trait WskWebActionsTests
                 response.statusCode shouldBe 200
                 response.header("Access-Control-Allow-Origin") shouldBe "*"
                 response.header("Access-Control-Allow-Methods") shouldBe "OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH"
-                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type"
+                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type, User-Agent"
```

Comment:
true.  not related to cli, per-se, but web apps making api requests having web app specific user agent headers should be supported.  see related question above https://github.com/apache/incubator-openwhisk/pull/2591#issuecomment-322465029